src/plugins/data/public/antlr/opensearch_ppl/code_completion.ts

```diff
@@ -0,0 +1,105 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+import { monaco } from '@osd/monaco';
+import { CursorPosition, AutocompleteResultBase } from '../shared/types';
+import { parseQuery } from '../shared/utils';
+import { openSearchPplAutocompleteData } from './opensearch_ppl_autocomplete';
+import { QuerySuggestion } from '../../autocomplete';
+import { IndexPattern, IndexPatternField } from '../../index_patterns';
+
+const fetchFieldSuggestions = (
```

Comment:
i think there can be a space in the language manager we where we register code completion stuff so that these files can be put in the query enhancements plugin.

cc: @ashwin-pc, @abbyhu2000 what do you think?